packages/react-ui-core/src/Collapsible/Collapsible.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class Collapsible extends Component {
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+    this.state = {
+      display: false,
+      toggleItems: this.props.toggleItems,
+    }
+  }
+  handleClick() {
```

Comment:
can you add a newline between methods, to make it more consistent with our current code? See [this file](https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Form/Field.js) for example.